ankisyncd/config.py

```diff
@@ -11,6 +11,16 @@
     os.path.join(dirname(dirname(realpath(__file__))), "ankisyncd.conf"),
 ]
 
+# Get values from ENV and update the config. To use this prepend `ANKISYNCD_`
+# to the uppercase form of the key. E.g, `ANKISYNCD_SESSION_MANAGER` to set
+# `session_manager`
+def load_from_env(conf):
+    logging.info("Loading/overriding config values from ENV")
+    for env in os.environ:
+        if env.startswith('ANKISYNCD_'):
+            config_key = env[10:].lower()
+            conf[config_key] = os.getenv(env)
+            logging.info(f"Setting {config_key} from ENV")
```

Comment:
F-strings are Python 3.6+, the server currently still supports Python 3.5 so you'll need to change this.

Reply:
Changed to `.format`